### PR TITLE
VSTS-1735 - Driver Education - AJAX Spinner

### PR DESCRIPTION
**Pull request by jessedoyle**
:art:

* To help overcome slow APIs, we need to have a loading animation
  to let the user know that "something's happening". Our old
  loading spinner doesn't aesthetically fit with the design of
  the driver education pages.
* Tentatively select a loading animation from here:
  http://tobiasahlin.com/spinkit/

SEE: https://amaabca.visualstudio.com/membership_backlog/_workitems?id=1735

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [X] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [X] Any applicable version numbers have been updated.
- [X] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [X] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [X] I have written a detailed PR message explaining what is being changed and why
- [X] I’ve linked to any relevant VSTS tickets
- [X] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design
- [ ] I've added new components to the style guide (or have a PR in to add them).

### Demo

loading_spinner

### CODE REVIEW

**Comment by DaveAMA** [comment]
📖

File: app/assets/stylesheets/layout_components/spinner.scss [hunk 1]
```diff
@@ -6,6 +6,33 @@
...
+    width: 70px;
+    text-align: center;
+  }
+}
+
+.spinner--bounce > div{
+  width: 18px;
+  height: 18px;
+  background-color: #333;
+
```
Comment (DaveAMA) [inline app/assets/stylesheets/layout_components/spinner.scss]: This extra line isn't needed

File: app/assets/stylesheets/layout_components/spinner.scss [hunk 2]
```diff
@@ -6,6 +6,33 @@
...
+    margin: 0 auto;
+    width: 70px;
+    text-align: center;
+  }
+}
+
+.spinner--bounce > div{
+  width: 18px;
+  height: 18px;
+  background-color: #333;
```
Comment (DaveAMA) [inline app/assets/stylesheets/layout_components/spinner.scss]: I would switch this colour to use our brand blue variable

File: app/assets/stylesheets/layout_components/spinner.scss (hunk 1 above)
Comment (DaveAMA) [inline app/assets/stylesheets/layout_components/spinner.scss]: this extra line isn't needed

**Comment by DaveAMA** [comment]
💩

File: app/assets/stylesheets/layout_components/spinner.scss (hunk 2 above)
Comment (DaveAMA) [reply app/assets/stylesheets/layout_components/spinner.scss]: $brand-blue-light

**Comment by jessedoyle** [comment]
:wave:

**Comment by DaveAMA** [comment]
🎨 🎨 🎨

**Pull request merged**